sway/desktop/output.c

```diff
@@ -321,82 +322,96 @@ static void render_container_simple_border_normal(struct sway_output *output,
 	struct wlr_box box;
 	float color[4];
 
-	// Child border - left edge
-	memcpy(&color, colors->child_border, sizeof(float) * 4);
-	color[3] *= con->alpha;
-	box.x = con->x;
-	box.y = con->y + 1;
-	box.width = con->sway_view->border_thickness;
-	box.height = con->height - 1;
-	render_rect(output->wlr_output, output_damage, &box, color);
-
-	// Child border - right edge
-	if (con->parent->children->length == 1 && con->parent->layout == L_HORIZ) {
-		memcpy(&color, colors->indicator, sizeof(float) * 4);
-	} else {
-		memcpy(&color, colors->child_border, sizeof(float) * 4);
-	}
-	color[3] *= con->alpha;
-	box.x = con->x + con->width - con->sway_view->border_thickness;
-	box.y = con->y + 1;
-	box.width = con->sway_view->border_thickness;
-	box.height = con->height - 1;
-	render_rect(output->wlr_output, output_damage, &box, color);
+	struct sway_container *workspace = container_parent(con, C_WORKSPACE);
+	int other_views = workspace->children->length - 1;
```

Comment:
What if the workspace contains a single container, and that container contains several children? Wouldn't you need to count all the views recursively?

Reply:
You are correct. I missed that case when testing. I'll update the code